src/chapters/functions/creating-functions.rst

```diff
@@ -0,0 +1,425 @@
+
+Since ``return`` terminates the function, we can leave out the ``else`` clause and have the same effect. This is because if ``n`` is even, the return statement in the ``if`` block will execute and the function will end. If ``n`` is odd, the ``if`` block will be skipped and the second return statement will execute.
+
+.. sourcecode:: js
+
+   function isEven(n) {
+      if (n % 2 === 0) {
+         return true;
+      }
+      return false;
+   }
+
+This updated version works exactly the same as our initial function. 
+
+Additionally, notice that the function returns ``true`` when ``n % 2 === 0`` returns ``true``, and it returns ``false`` when ``n % 2 === 0`` returns ``false``. In other words, the return value is *exactly the same* as the value of ``n % 2 === 0``. This means that we can simplify the function even further by returning the value of this expression.
+
+.. sourcecode:: js
+
+   function isEven(n) {
+      return n % 2 === 0;
+   }
+
+This version of ``isEven`` is better than the first two, not because it is shorter (shorter isn't always better), but because it is simpler to read. We don't have to break down the conditional logic to see what is being returned.
+
+Most boolean functions can be written so that they return the value of a boolean expression, rather than explicitly returning ``true`` or ``false``. 
+
+Parmeters and Arguments
+-----------------------
+
+.. index::
+   single: function; argument
+   single: function; parameter
+
+Over the past two sections, we introduced two function-related concepts that are very similar, and are often confusing to distinguish: arguments and parameters. If it isn't clear to you what the difference between these two are, then you are not alone. We will attempt to clear that up right now, however.
+
+The easiest way to talk about the difference between arguments and parameters is by referring to an example.
+
+.. admonition:: Example
+
+   The function ``hello`` takes a single value, which we expect to be a person's name, and returns a message that greets that person. 
+
+   .. sourcecode:: js
+      :linenos:
+
+      function hello(name) {
+         return `Hello, ${name}!`;
+      }
+
+      console.log(hello("Lamar"));
+
+   **Output**
+
+   ::
+
+      Hello, Lamar!
+
+
+In this example, ``name`` is a **parameter**. It is part of the function definition, and *behaves like a variable* that exists only within the function.
+
+The value ``"Lamar"`` that is use when we invoke the function on line 5 is an **argument**. It is a *specific value* that is used during the function call. 
+
+You should think of the difference between an parameter and an argument as being the same as that between a variable and a value. A variable *refers to* a specific value, just like a parameter *refers to* a specific argument when a function is called. Like a value, a variable is a concrete piece of data.
```

Comment:
Typos: 'a parameter' rather than 'an parameter'.  Also, "...an argument is a concrete piece of data."

How about: "The difference between a *parameter* and an *argument* is the same as the difference between a *variable* and a *value*. A variable *refers to* a specific value, just like a parameter *refers to* a specific argument when a function is called. Like a value, an argument is a concrete piece of data."